content/concepts/the-rippled-server/peer-protocol.md

```diff
@@ -17,152 +26,34 @@ ip = 0.0.0.0
-| `uptime`     | Number                   | The number of seconds the server has been connected to this peer. |
-| `version`    | String                   | The `rippled` version number the peer reports to be using. |
+When a server first starts up, it generates a _node key pair_ to use to identify itself in peer protocol communications. The server uses its key to sign all its peer protocol communications. This makes it possible to reliably identify and verify the integrity of messages from another server in the peer-to-peer network even if that server's messages are being relayed by untrusted peers. A node key pair is similar to a validator's ephemeral key pair, but validation key pairs are treated separately from node key pairs.
 
-#### Example
+The node key pair is saved in the database and reused when the server restarts. If you delete the server's databases, it creates a new node key pair, effectively coming online with a different identity. To reuse the same key pair even if the databases are deleted, you can configure the server with a `[node_seed]` stanza. To generate a value suitable for use in the `[node_seed]` stanza, use the [validation_create method][].
 
-Request:
+The node key pair also identifies other servers [clustered](clustering.html) with this one. If you have a cluster of servers, you should configure each server in the cluster with a unique `[node_seed]` setting. For more information on setting up a cluster, see [Cluster `rippled` Servers](cluster-rippled-servers.html).
 
-<!-- MULTICODE_BLOCK_START -->
+## Private Peers
 
-*HTTP*
+You can configure a `rippled` server to act as a "private" server to keep its IP address hidden from the general public. This can be a useful precaution against denial of service attacks and intrusion attempts on important `rippled` servers such as trusted validators. To participate in the peer-to-peer network, a private server must be configured to connect to at least one non-private server, which relays its messages to the rest of the network.
 
-```
-GET https://s1.ripple.com:51235/crawl
-```
+Configuring a server as a private server has several effects:
 
-*cURL*
+- The server does not make outgoing connections to other servers in the peer-to-peer network unless it has been explicitly configured to connect to those servers.
+- The server does not accept incoming connections from other servers unless it has been explicitly configured to accept connections from those servers.
+- The server asks its direct peers not to reveal its IP address in untrusted communications, including the [peer crawler API response](peer-crawler.html). This does not affect trusted communications such as the [peers admin method][peers method].
 
-```
-curl -k https://s1.ripple.com:51235/crawl
-```
+    Servers configured as validators do this even if they aren't configured as private peers. This helps protect validators from being overloaded by denial of service attacks. [New in: rippled 1.2.1][]
```

Comment:
replace "do this" with something more specific, so the sentence works standalone. Maybe something like:  "Validators are configured as private by default (new in 1.2.1), this helps protect..."